docs/guide/displaying_children_of_a_content_item.md

```diff
@@ -49,7 +49,43 @@ class LocationChildrenQueryType implements QueryType
 }
 ```
 
-Next, in your [standard view configuration](../guide/content_rendering.md#configuring-views-the-viewprovider) file, under `content_view`, add a section that indicates when this Controller will be used. It is similar to regular view config, but contains additional information:
+To use the Query Type with a Content query Field, add the Field to your Folder Content Type's definition.
+
+Select "Location children" as the Query type. Provide the `parentLocationId` parameter
+that the Query type requires:
+
+```
+parentLocationId: '@=mainLocation.id'
+```
+
+To customize the display template, in your [standard view configuration](../guide/content_rendering.md#configuring-views-the-viewprovider) file,
+under `content_view`, add a section that indicates the matcher and template to use: 
+
+``` yaml
+content_query_field:
+    folder_children:
+        match:
+            Identifier\ContentType: folder
+            Identifier\FieldDefinition: children
```

Comment:
There seems to be a problem with this matcher (on master):

```
An exception has been thrown during the rendering of a template ("Invalid matcher class 'eZ\Publish\Core\MVC\Symfony\Matcher\ContentBased\Identifier\FieldDefinition'") in "@EzSystemsEzPlatformQueryFieldType/fieldtype/field_view.html.twig".
```

Any chance it's the doc that is missing something? ping @bdunogier

Reply:
I actually noticed that during the demos last week. I was sure I had tested it successfully, but I guess not. We're gonna have to improve the View Matching system to make it work.

Reply:
I have created an issue for it: https://jira.ez.no/browse/EZP-31288. I'll try to find a fix for this.

Reply:
All the fixes are in review / finalization phase:
- https://github.com/ezsystems/ezplatform-query-fieldtype/pull/15 (for Platform 2.x)
- https://github.com/ezsystems/ezplatform-query-fieldtype/pull/16 (for Platform 3.x)
- https://github.com/ezsystems/ezpublish-kernel/pull/2916 (for Platform 3.x, required to respect the documented syntax)

Reply:
https://github.com/ezsystems/ezplatform-query-fieldtype/pull/15 and https://github.com/ezsystems/ezplatform-query-fieldtype/pull/16 were merged, the field type should now behave as expected in all versions (no tag yet).